src/orders/components/OrderDraftListDeleteButton/OrderDraftListDeleteButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, Tooltip, TrashBinIcon } from "@saleor/macaw-ui/next";
+import React, { forwardRef, ReactNode, useState } from "react";
+
+interface OrderDraftListDeleteButtonProps {
+  onClick: () => void;
+  children: ReactNode;
+}
+
+export const OrderDraftListDeleteButton = forwardRef<
+  HTMLButtonElement,
+  OrderDraftListDeleteButtonProps
+>(({ onClick, children }, ref) => {
+  const [isTooltipOpen, setIsTooltipOpen] = useState(false);
+
+  return (
+    <Tooltip open={isTooltipOpen}>
+      <Tooltip.Trigger>
+        <Button
+          ref={ref}
+          onMouseOver={() => {
+            setIsTooltipOpen(true);
+          }}
+          onMouseLeave={() => {
+            setIsTooltipOpen(false);
+          }}
+          onClick={onClick}
+          icon={<TrashBinIcon />}
+          variant="secondary"
+          data-test-id="delete-categories-button"
```

Comment:
**suggestion**
I don't think those are categories :D
Perhaps this button can be abstracted (ButtonWithTooltip or something)
If not you can just change the data-test-id